test.py

```diff
@@ -10,12 +10,10 @@ class TestQIF(unittest.TestCase):
     def setUp(self):
         pass
 
-
     def test_get_qif_name(self):
 
-        name = qif.get_qif_name('01 Jan 10', '13 Apr 12')
-        self.assertEqual(name, '2010.01.01 - 2012.04.13.qif')
-
+        name = qif.get_qif_name('1 Jan 10', '13 Apr 12')
+        self.assertEqual(name, '2010.1.1 - 2012.4.13.qif')
```

Comment:
Could this be changed back to the exiting functionality, where it outputs `2010.01.01` instead of `2010.1.1`?